contracts/proof/RLP.sol

```diff
@@ -33,8 +33,8 @@ library RLP {
 
  function next(Iterator memory self) internal pure returns (RLPItem memory subItem) {
      require(hasNext(self));
-     var ptr = self._unsafe_nextPtr;
-     var itemLength = _itemLength(ptr);
+     uint256 ptr = self._unsafe_nextPtr;
```

Comment:
Do line# 29 needs to change to uint256 ?

Reply:
uint and int are aliases for uint256 and int256, respectively.

Reply:
Yes because _itemLength takes ptr as input which is uint256.